### DISCUSSION TITLE
How can I prevent the inclusion of the %(AdditionalDependencies) tag?

### DISCUSSION THREAD

**Discussion by hannes-harnisch** [premake/premake-core#1733] [Q&A] [answered]
I wanna reduce the number of Win32 libraries that get linked into my program by default, which is caused by the %(AdditionalDependencies) tag under the linker settings. This tag causes project default libraries to be linked as well, most of which I don't need.

I couldn't find an option to prevent the inclusion of these default libraries (except non-portable flags). Is there a simple way to do this that I missed?

**Comment by starkos** [comment, marked as answer]
At the moment, there is no provided way to turn that off. Wouldn't be a difficult thing for someone to add a `systemLibraries 'off'` switch though: copy one of the existing switches in `_premake_init.lua`, use it in `vcxproj.lua` and add a test to cover it.
